internal/dbtest/query_test.go

```diff
@@ -943,3 +945,128 @@ func TestQuery(t *testing.T) {
 		}
 	})
 }
+
+func TestAlterTable(t *testing.T) {
+	needsAlterTable := needs(feature.AlterTableQuery, "ALTER TABLE")
+	type Model struct {
+		ID     int64
+		Old    string
+		Active bool
+	}
+
+	cases := []TestCase{
+		{
+			"rename column", needsAlterTable,
+			func(db *bun.DB) schema.QueryAppender {
+				return db.NewAlterTable().Model((*Model)(nil)).
+					RenameColumn().Column("old").To("new")
+			},
+		},
+		{
+			"invalid model", needsAlterTable,
+			func(db *bun.DB) schema.QueryAppender {
+				return db.NewAlterTable().Model(1)
+			},
+		},
+		{
+			"rename table", needsAlterTable,
+			func(db *bun.DB) schema.QueryAppender {
+				return db.NewAlterTable().Model((*Model)(nil)).
+					Rename().To("new_models")
```

Comment:
Why not `RenameTable("new_name")`?

Reply:
You're right, that's actually [what I started with](https://github.com/uptrace/bun/pull/726/commits/f8ff602c226a80b3eb4d846bd6aeb062a22ae3e7), but I wasn't sure if it wasn't more consistent with the rest of the API to mirror actual the SQL keywords.  
I agree that this is much more readable and explicit though.